src/SIM/XMLInputBase.C

```diff
@@ -56,7 +56,7 @@ struct IncludeInjector : public tinyxml2::XMLVisitor
           return false;
       }
       currElem->InsertEndChild(inc_doc.RootElement()->DeepClone(&new_doc));
-      include_found = true;
+      include_found = include_processed = true;
     } else {
       e = new_doc.NewElement(elem.Name());
```

Comment:
The temporary `e` is defined outside this block but only (afics) used inside it. Wouldn't that cause some static code check warnings?

Reply:
possibly. reduced the scope.